lib/rules/aria-roledescription.json

```diff
@@ -0,0 +1,13 @@
+{
+	"id": "aria-roledescription",
+	"selector": "[aria-roledescription]",
+	"excludeHidden": false,
+	"tags": ["cat.aria", "best-practice"],
```

Comment:
Need to engage with the Jedi Council to determine whether this should be categorized under WCAG 4.1.2

My initial reaction is that it should because if used incorrectly, it means that the role is not being described correctly.

Reply:
I'd say keep the tags consistent with what it was before we split out: `"cat.aria", "wcag2a", "wcag412"`